openquake/hazardlib/imt.py

```diff
@@ -106,6 +106,14 @@ def PGD():
     return IMT('PGD')
 
 
+def EAS(frequency):
```

Comment:
I guess this assumes the fact that the table of coefficient will be based on periods rather than frequencies, right?

Reply:
No, this assumes the table of coefficients to be based on frequencies.

Reply:
The coeffs table consists of frequencies and not periods. Also, it makes sense to express EAS in frequency.
<img width="444" alt="Screenshot 2021-11-26 at 3 04 48 PM" src="https://user-images.githubusercontent.com/78085634/143559246-632372ed-9ba3-4292-b9a4-92bd68ce3f30.png">
.